autoload/neoformat/formatters/sbt.vim

```diff
@@ -0,0 +1,11 @@
+function! neoformat#formatters#sbt#enabled() abort
+    return ['scalafmt']
+endfunction
+
+function! neoformat#formatters#sbt#scalafmt() abort
+    return {
+        \ 'exe': 'scalafmt',
+        \ 'args': ['--stdin', '--assume-filename', 'foo.sbt'],
```

Comment:
I could.  But if for some reason (and I can't think of a good one why one might do this), you were editing a file that didn't end in `.sbt`, but had set the filetype to `sbt` directly or via a modeline or something, then using the actual filename as the argument would cause a problem.  The execution errors out and the buffer gets overwritten with nothing.

I could make the dummy filename something like `dummy.sbt` to make it clearer, and/or add a comment?

Reply:
I think a comment should suffice